srml/assets/src/lib.rs

```diff
@@ -96,6 +239,7 @@ decl_event!(
 	}
 );
 
+/// BLAH
```

Comment:
should be removed ?
```suggestion
```
no doc here seems fine to me as every storages are documented

Reply:
yes i've removed that now, thanks